libmamba/src/api/update.cpp

```diff
@@ -187,17 +228,17 @@ namespace mamba
         );
 
 
-        auto execute_transaction = [&](MTransaction& transaction)
+        auto execute_transaction = [&](MTransaction& trans)
```

Comment:
Why shortening such a clear name? 🥲

Reply:
Because of the shadowing warnings 🥲
We can also do something like `inner_transaction` but it's longer...

Reply:
Oh ok, no it's fine as is 👍🏽 I didnt know about the shadowing warnings.
I personally dont like warnings about shadowing because shadowing is an excellent way to make the local code clear without impacting the global code, but most people disagree hahaha